2-improved.js

```diff
@@ -0,0 +1,13 @@
+let first_num = 300;
+let second_num = 200;
+let total = total_list(first_num + second_num);
+
+function total_list() {
```

Comment:
- Use `camelCase` in JavaScript
- Better `const totalList = (...args) => {` syntax